uwblueprint/community-fridge-kw
# feat: volunteer pending email after verification

## Brief description. What is this change? 

### [Pending Email After Verification](https://www.notion.so/uwblueprintexecs/Pending-Email-After-Verification-bd7f5305c10a44abb540a04335609217)



## Implementation description. How did you make this change?
* Created `sendEmailVolunteerPending()` with [email copy](https://docs.google.com/document/d/1U_XHVW4ZXldruej4oA9pr0lLnp_nIOJwTgmLRHB7KFA/edit#bookmark=id.bbmgl2pqqj4m) in `authservices.tsx`
* Call `sendEmailVolunteerPending` in `verifyEmail`, if `response.emailVerified` and user is `volunteer`
   * Call `getUserByEmail` to get user’s role
* Updated `/services/interfaces/authService.ts`


## Steps to test
1. Create volunteer account and verify email
2. New email should be sent shortly after (had to take down prod to test)
<img width="800" alt="Screen Shot 2022-03-08 at 10 21 52 PM" src="https://user-images.githubusercontent.com/69697180/157367325-f1913921-cff7-47c2-b7ae-0527c519da8e.png">
<img height="500" alt="Screen Shot 2022-03-08 at 10 21 52 PM" src="https://user-images.githubusercontent.com/69697180/157367334-0b1f8c71-dedc-4c4b-832a-b3cbdf2cbacd.jpeg">




## Checklist
